sale_ux/models/sale_order_line.py

```diff
@@ -8,10 +8,6 @@
 class SaleOrderLine(models.Model):
     _inherit = 'sale.order.line'
 
```

Comment:
After this change there was introduces two pylint errors

> sale_ux/models/sale_order_line.py:5:1: F401 'odoo.fields' imported but unused
sale_ux/models/sale_order_line.py:12:5: E303 too many blank lines (2)

Please fix